<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SequenciedSettingsWidget</class>
 <widget class="QWidget" name="SequenciedSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* QWidget 背景与全局字体 */
QWidget {
    background-color: #e6f2ff; /* 浅蓝色背景 */
    color: #002b36;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 14px;
}

/* QLabel 标题文字 */
QLabel {
    color: #003366;
    font-weight: bold;
}

/* QPushButton 样式 */
QPushButton {
    background-color: #cce6ff;
    border: 1px solid #99ccff;
    border-radius: 4px;
    color: #003366;
}

QPushButton:hover {
    background-color: #b3daff;
}

QPushButton:pressed {
    background-color: #99ccff;
}

/* QSlider 样式 */
QSlider::groove:horizontal {
    border: 1px solid #99ccff;
    height: 6px;
    background: #ffffff;
    border-radius: 3px;
}

QSlider::handle:horizontal {
    background: #66ccff;
    border: 1px solid #3399ff;
    width: 14px;
    height: 14px;
    margin: -4px 0;
    border-radius: 7px;
}

QSlider::add-page:horizontal {
    background: #e6f2ff;
}

QSlider::sub-page:horizontal {
    background: #99ccff;
}

/* QSpinBox 样式 */
QSpinBox {
    background-color: #ffffff;
    border: 1px solid #99ccff;
    border-radius: 4px;
    padding: 2px 4px;
    color: #003366;
}

QSpinBox::up-button, QSpinBox::down-button {
    subcontrol-origin: border;
    width: 16px;
    background-color: #cce6ff;
    border-left: 1px solid #99ccff;
}

QSpinBox::up-button:hover, QSpinBox::down-button:hover {
    background-color: #b3daff;
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="image_displayerWidgets_left" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="image_displayerWidgets" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3"/>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_reimport">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>re_import</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Animation length(ms)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="animation_spinbox">
        <property name="maximum">
         <number>100000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Switching Trigger(ms)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="switching_trigger">
        <property name="maximum">
         <number>100000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="animate_type"/>
      </item>
      <item>
       <widget class="QLabel" name="clicked_label">
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
